### PR TITLE
Update manifest for signed builds.

### PR DESCRIPTION
**Pull request by anpaz**

### CODE REVIEW

File: build/manifest.ps1 [hunk 1]
```diff
@@ -1,17 +1,39 @@
...
 & "$PSScriptRoot/set-env.ps1"
 
-@{
+$artifacts = @{
     Packages = @(
         "Microsoft.Quantum.Research.Chemistry",
         "Microsoft.Quantum.Research.Characterization",
         "Microsoft.Quantum.Research"
-    );
+    ) | ForEach-Object { Join-Path $Env:NUGET_OUTDIR "$_.$Env:NUGET_VERSION.nupkg" };
```
Comment (rmshaffer) [inline build/manifest.ps1:25]: I believe `"Microsoft.Quantum.Research.Simulation"` is missing here.

Comment (anpaz) [reply build/manifest.ps1:25]: Yes, thanks for catching that. A few of them were missing indeed.

**Pull request merged**